string ip
uint32 port
string drone_type
---
bool success
string message
uint32 id
